.gitmodules

```diff
@@ -118,3 +118,8 @@
 	path = Sming/Libraries/MCP_CAN_lib
 	url = https://github.com/coryjfowler/MCP_CAN_lib.git
 	ignore = dirty
+[submodule "Sming/Components/libsodium/libsodium"]
```

Comment:
Should go at top (e.g. line 24) and rename to "libsodium"

Reply:
I put it under esptool2, as slaff proposed.